src/test/java/io/vertx/ext/cluster/infinispan/test/InfinispanClusterWideMapTest.java

```diff
@@ -57,4 +69,127 @@ protected void clusteredVertx(VertxOptions options, Handler<AsyncResult<Vertx>>
   protected ClusterManager getClusterManager() {
     return new InfinispanClusterManager();
   }
+
+  @Test
+  @Repeat(times = 10)
```

Comment:
good idea

Reply:
Actually a left-over from my local testing. Do we want to keep it?